tests in sdist are incomplete

### Bug description

The sdist package at PyPI does not contain complete tests.  There are few `tests/test_*.py` files there, but many other are missing.  The `tox.ini` file is missing too.

This causes tests failure.


### Configuration

```ini
NA
```


### Command used

```shell
pytest
```


### Pylint output

```
__________________ ERROR collecting tests/test_functional.py ___________________
/usr/lib/python3.9/vendor-packages/_pytest/runner.py:341: in from_call
    result: Optional[TResult] = func()
/usr/lib/python3.9/vendor-packages/_pytest/runner.py:372: in <lambda>
    call = CallInfo.from_call(lambda: list(collector.collect()), "collect")
/usr/lib/python3.9/vendor-packages/_pytest/python.py:531: in collect
    self._inject_setup_module_fixture()
/usr/lib/python3.9/vendor-packages/_pytest/python.py:545: in _inject_setup_module_fixture
    self.obj, ("setUpModule", "setup_module")
/usr/lib/python3.9/vendor-packages/_pytest/python.py:310: in obj
    self._obj = obj = self._getobj()
/usr/lib/python3.9/vendor-packages/_pytest/python.py:528: in _getobj
    return self._importtestmodule()
/usr/lib/python3.9/vendor-packages/_pytest/python.py:617: in _importtestmodule
    mod = import_path(self.path, mode=importmode, root=self.config.rootpath)
/usr/lib/python3.9/vendor-packages/_pytest/pathlib.py:567: in import_path
    importlib.import_module(module_name)
/usr/lib/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1030: in _gcd_import
    ???
<frozen importlib._bootstrap>:1007: in _find_and_load
    ???
<frozen importlib._bootstrap>:986: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:680: in _load_unlocked
    ???
/usr/lib/python3.9/vendor-packages/_pytest/assertion/rewrite.py:186: in exec_module
    exec(co, module.__dict__)
tests/test_functional.py:31: in <module>
    for t in get_functional_test_files_from_directory(FUNCTIONAL_DIR)
pylint/testutils/functional/find_functional_tests.py:42: in get_functional_test_files_from_directory
    _check_functional_tests_structure(Path(input_dir), max_file_per_directory)
pylint/testutils/functional/find_functional_tests.py:105: in _check_functional_tests_structure
    for file_or_dir in walk(directory):
pylint/testutils/functional/find_functional_tests.py:82: in walk
    parent_dir_files = _get_files_from_dir(path, violations)
pylint/testutils/functional/find_functional_tests.py:72: in _get_files_from_dir
    files_without_leading_underscore = list(
pylint/testutils/functional/find_functional_tests.py:72: in <genexpr>
    files_without_leading_underscore = list(
/usr/lib/python3.9/pathlib.py:1160: in iterdir
    for name in self._accessor.listdir(self):
E   FileNotFoundError: [Errno 2] No such file or directory: '$(BUILD_DIR)/tests/functional'
```


### Expected behavior

Tests pass

### Pylint version

```shell
3.0.2
```


### OS / Environment

_No response_

### Additional dependencies

_No response_

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,1 +1,11 @@
 include README.rst
+include tox.ini
+graft doc
+graft examples
+graft script
+graft tests
+prune doc/_build
+prune tests/.benchmarks
+prune tests/.pylint_primer_tests
+global-exclude __pycache__
+global-exclude *.py[co]

